plugin/editorconfig.vim

```diff
@@ -459,11 +459,19 @@ function! s:SpawnExternalParser(cmd) " {{{2
         " In Windows, 'shellslash' also changes the behavior of 'shellescape'.
         " It makes 'shellescape' behave like in UNIX environment. So ':setl
         " noshellslash' before evaluating 'shellescape' and restore the
-        " settings afterwards.
-        let l:old_shellslash = &l:shellslash
-        setlocal noshellslash
+        " settings afterwards when 'shell' does not contain 'sh' somewhere.
+        if ((has('win32') || has('win64')) && empty(matchstr(&shell, 'sh')))
```

Comment:
@hail2u I think `has('win64')` here is unnecessary, since `win32` includes `win64` (see `:h feature-list` for details). Do you agree?